### DISCUSSION TITLE
How can I add and translate new language in Floorp Crowdin?

### DISCUSSION THREAD

**Discussion by dodog** [Floorp-Projects/Floorp#1459] [Q&A]
How can I add and translate a new language in Floorp Crowdin?

I changed the language to Slovak in the Floorp browser settings, but nothing happened. In Crowdin, Slovak is not available for the Floorp browser. I tried to contact the administrator of the translation project, but no response. Could someone help me to fix this?

**Comment by Pikku-a** [comment]
+1
Same problem, but with different language.

**Comment by dodog** [comment]
I gave up. I have sent many language requests on Crowdin, but @surapunoyousei ignored them all. 
It seems that this is just a one-man project.

**Reply by CutterKnife** [reply]
He might just not have noticed it yet. Why not open an issue [here](https://github.com/Floorp-Projects/f18n-central/issues)?

**Reply by dodog** [reply]
30 requests in the last year? No response.
Most of them in Crowdin. Also DMs on Crowdin. No response.
Also on [Floorp Github Unified-l10n-central](https://github.com/Floorp-Projects/Unified-l10n-central/issues/43
) - no response.  Also in [Floorp Github](https://github.com/Floorp-Projects/Floorp/issues/1479).
Maybe you could send him a message to look at the requests in Crowdin?

**Reply by CutterKnife** [reply]
I sent pull requests to some of Floorp-Projects' repositories, but some of them went unanswered, so maybe he doesn't check Crowdin that often.
Just to confirm, is Slovak available in Firefox?

**Reply by dodog** [reply]
Yes, of course.

**Reply by CutterKnife** [reply]
The language files for Floorp 12 are hosted on [f18n-central](https://github.com/Floorp-Projects/f18n-central).
If you open an issue here, he'll likely notice it within a few months. (He's very busy, so I can't guarantee it.)
